pkgs/development/python-modules/pytest-factoryboy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, pythonPackages }:
+
+buildPythonPackage rec {
+  pname = "pytest-factoryboy";
+  version = "2.0.3";
+
+  src = fetchFromGitHub {
+    owner = "pytest-dev";
+    repo = "pytest-factoryboy";
+    rev = version;
+    sha256 = "0m1snyybq2k51khlydhisq300vzys897vdbsicph628iran950hn";
+  };
+
+  # The project uses tox, which we can't. So we simply run pytest manually.
+  checkInputs = with pythonPackages; [
+    pytest
+    pytestcache
+    pytestcov
+    pytestpep8
+    mock
+  ];
+  checkPhase = ''
+    pytest -vv -l --pep8 --ignore docs
+  '';
```

Comment:
Using pytestCheckHook, this can then become
```nix
pytestFlagsArray = [
  "--pep8"
  "--ignore=docs"
];
```
I also suggest removing the "-vv" and "-l" flags, which are for extra verbosity and showing local variables on failures respectively. They are useful while debugging, but not required for the Nix build.

Reply:
Done.